### ISSUE TITLE
Desktop Shortcut issues - .ico file

### ISSUE THREAD

**Issue by pehammer** [JaneaSystems/PowerToys-DevProjects#10]
When pinning a "Project", it creates a shortcut as well as a .ico file on my desktop, and I couldn't initially tell which I should click. Clicking the shortcut launches an unexpected terminal instance.

![image](https://github.com/JaneaSystems/PowerToys-DevProjects/assets/112589934/855312e9-f5b3-4135-8e1d-378fe675788f)

**Comment by noraa-junker** [comment]
Yes, I would suggest, if the ico file is needed, it should be saved somewhere in Appdata

**Comment by pehammer** [comment]
The oneDrive checkmark adds to the confusion
